smoke-tests/matrix/src/main/java/io/opentelemetry/smoketest/matrix/ReceiveSendGreetingServlet.java

```diff
@@ -0,0 +1,50 @@
+package io.opentelemetry.smoketest.matrix;
+
+import com.google.gson.Gson;
+import io.opentelemetry.smoketest.matrix.model.GreetingWithHeader;
+import io.opentelemetry.smoketest.matrix.util.StreamTransferUtil;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.stream.Collectors;
+
+public class ReceiveSendGreetingServlet extends HttpServlet {
```

Comment:
Could we rename it `EchoServlet`?

Reply:
Could do servlet just simply read the body to a buffer and write it back with the same content type? 

This way we don't need dep on Gson and additional classes. To test response heders capture the servlet could also send some test response header.

Reply:
@pavolloffay  made below changes. 
1. renamed the servlet to EchoServlet
2. copying request as response . Header going as test headers.